### PR TITLE
Feature/storage/blob tags75

### PR DESCRIPTION
**Pull request by seanmcc-msft**

### CODE REVIEW

File: sdk/storage/Azure.Storage.Blobs/swagger/readme.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 ## Configuration
 ``` yaml
 # Generate blob storage
-input-file: https://raw.githubusercontent.com/Azure/azure-rest-api-specs/storage-dataplane-preview/specification/storage/data-plane/Microsoft.BlobStorage/preview/2020-02-10/blob.json
+input-file: C:\Users\seanmcc\git\azure-rest-api-specs\specification\storage\data-plane\Microsoft.BlobStorage\preview\2020-04-08\blob.json
```
Comment (kasobol-msft) [inline sdk/storage/Azure.Storage.Blobs/swagger/readme.md:7]: local path.

File: sdk/storage/generate.ps1 [hunk 2]
```diff
@@ -5,16 +5,16 @@ npm install -g autorest@beta
...
 
-cd $PSScriptRoot/Azure.Storage.Queues/swagger/
-autorest --use=$PSScriptRoot/Azure.Storage.Common/swagger/Generator/ --verbose
+#cd $PSScriptRoot/Azure.Storage.Queues/swagger/
+#autorest --use=$PSScriptRoot/Azure.Storage.Common/swagger/Generator/ --verbose
 
-cd $PSScriptRoot/Azure.Storage.Files.DataLake/swagger/
-autorest --use=$PSScriptRoot/Azure.Storage.Common/swagger/Generator/ --verbose
+#cd $PSScriptRoot/Azure.Storage.Files.DataLake/swagger/
+#autorest --use=$PSScriptRoot/Azure.Storage.Common/swagger/Generator/ --verbose
```
Comment (kasobol-msft) [inline sdk/storage/generate.ps1:18]: this should be reverted.

**Review by kasobol-msft — APPROVED** [review]

File: sdk/storage/Azure.Storage.Blobs/swagger/readme.md [hunk 3]
```diff
@@ -4,7 +4,7 @@
 ## Configuration
 ``` yaml
 # Generate blob storage
-input-file: https://raw.githubusercontent.com/Azure/azure-rest-api-specs/storage-dataplane-preview/specification/storage/data-plane/Microsoft.BlobStorage/preview/2020-02-10/blob.json
```
Comment (jaschrep-msft) [inline sdk/storage/Azure.Storage.Blobs/swagger/readme.md:7]: revert?

**Comment by seanmcc-msft** [comment]
I'm going to remove all the local paths in this PR before merging, they are currently there by design.

**Pull request merged**